fastjet/internal/predicates/incircle.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2017 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package predicates
+
+import (
+	"math/big"
+)
+
+const (
+	// macheps is the machine epsilon aka unit roundoff
+	// it is the upper bound on relative error due to rounding
+	macheps = 2.22e-16
```

Comment:
I would perhaps add some reference as to where it is coming from.
https://en.wikipedia.org/wiki/Machine_epsilon ?

and also explain the value (2^-52)